### ISSUE TITLE
User defined structures

### ISSUE THREAD

**Issue by pvasilek** [AlexanderBagel/ProcessMemoryMap#9]
День добрый,

планируете ли сделать навигацию по памяти с использованием известных структур, а также объявленных пользователем?
Чтобы, например, можно было начать с какой-то точки в памяти, отобразить на нее структуру данных, а при наличии в структуре данных указателей на другие структуры - обеспечить навигацию по такому графу.

**Comment by AlexanderBagel** [comment]
Да, планирую, но я хотел это начать реализовывать только когда до PDB дойду ибо хочется вообще всю работу со структурами перенести на единый движок а не как сейчас, когда они захардкожены прямо в коде

**Comment by recoil24** [comment]
Блин первый раз нашёл на github'e земляка который так жёстко разбирается в kernel среде. Респект!
